gnssanalysis/gn_download.py

```diff
@@ -268,8 +292,13 @@ def check_n_download_url(url, dwndir, filename=False):
         out_f = _Path(dwndir) / filename
         download_url(url, out_f)
 
+    if uncomp:
+        _subprocess.run(["uncompress", f"{out_f}"])
```

Comment:
I believe that `uncompress` only works for ".Z" style unix compression and so will fail for eg ".gz" or ".lz" or ".zip" files (I think?) and if you point it at a non-compressed URL and ask for decompression you will also get a failure. Because we don't have `check=True` set in the `subprocess.run` call this will be a silent failure that won't break the python but it's still maybe something to think about and maybe non-intuitive for a user.